src/System.Text.RegularExpressions/tests/Regex.Replace.Tests.cs

```diff
@@ -224,6 +224,31 @@ public void Replace_NoMatch()
             Assert.Same(input, Regex.Replace(input, "no-match", new MatchEvaluator(MatchEvaluator1)));
         }
 
+        [Theory]
+        [InlineData(RegexOptions.None)]
+        [InlineData(RegexOptions.RightToLeft)]
+        public void Replace_MatchEvaluatorReturnsNullOrEmpty(RegexOptions options)
+        {
+            string input = "abcde";
```

Comment:
Not sure why input is defined here in a local but the pattern is defined inline in the Replace call... seems like you could just move "abcde" to be the arg